src/imgui/ImGuiReverseBar.cc

```diff
@@ -190,22 +190,52 @@ void ImGuiReverseBar::showMenu(MSXMotherBoard* motherBoard)
 				ImGui::TextUnformatted("No replays found"sv);
 			} else {
 				ImGui::TextUnformatted("Select replay"sv);
-				im::ListBox("##select-replay", [&]{
-					for (const auto& [fullName_, displayName_, _] : replayNames) {
+				int flags = ImGuiTableFlags_RowBg |
+					ImGuiTableFlags_BordersV |
+					ImGuiTableFlags_BordersOuter |
+					ImGuiTableFlags_Resizable |
+					ImGuiTableFlags_Sortable |
+					ImGuiTableFlags_Hideable |
+					ImGuiTableFlags_Reorderable |
+					ImGuiTableFlags_ContextMenuInBody |
+					ImGuiTableFlags_SizingStretchProp;
+				im::Table("##select-replay", 2, flags, {}, [&]{
+					for (const auto& [fullName_, displayName_, ftime_] : replayNames) {
 						const auto& fullName = fullName_; // clang workaround
-						const auto& displayName = displayName_; // clang workaround
-						if (ImGui::Selectable(displayName.c_str())) {
-							manager.executeDelayed(makeTclList("reverse", "loadreplay", fullName));
-						}
-						im::PopupContextItem([&]{
+						if (ImGui::TableNextColumn()) {
+							const auto& displayName = displayName_; // clang workaround
+							if (ImGui::Selectable(displayName.c_str(), false, ImGuiSelectableFlags_SpanAllColumns | ImGuiSelectableFlags_AllowOverlap)) {
+								manager.executeDelayed(makeTclList("reverse", "loadreplay", fullName));
+							}						im::PopupContextItem([&]{
 							if (ImGui::MenuItem("delete")) {
 								confirmCmd = makeTclList("file", "delete", fullName);
 								confirmText = strCat("Delete replay '", displayName, "'?");
 								openConfirmPopup = true;
 							}
 						});
-					}
-				});
+
+						}
+						if (ImGui::TableNextColumn()) {
+							const auto& ftime = ftime_; // clang workaround
+
+							// Convert file_time_type to system_clock::time_point (the standard clock since epoch)
+							auto sctp = std::chrono::time_point_cast<std::chrono::system_clock::duration>(
+									ftime - std::filesystem::file_time_type::clock::now() +
+									std::chrono::system_clock::now());
+
+							// Convert system_clock::time_point to time_t (time since Unix epoch in seconds)
+							std::time_t cftime = std::chrono::system_clock::to_time_t(sctp);
+
+							// Convert time_t to local time (broken-down time in the local time zone)
+							std::tm* local_time = std::localtime(&cftime);
```

Comment:
Maybe add some comment here that we're using a workaround, because (IIRC) `clock_cast`, `zoned_time`, and `std::format` are only supported starting from gcc-13.

Reply:
Will do.